src/CalcViewModel/Common/CalculatorButtonUser.h

```diff
@@ -161,55 +161,14 @@ public
         BINPOS62 = (int)CM::Command::CommandBINPOS62,
         BINPOS63 = (int)CM::Command::CommandBINPOS63,
         BINEND = (int)CM::Command::CommandBINEDITEND,
-        Hyp = (int)CM::Command::CommandHYP
-    };
+        Hyp = (int)CM::Command::CommandHYP,
 
-    // This contains list of functions whose usage we are tracelogging
-public
-    enum class FunctionLogEnum
-    {
-        Invert = (int)CM::Command::CommandREC,
-        Sqrt = (int)CM::Command::CommandSQRT,
-        Percent = (int)CM::Command::CommandPERCENT,
-        Negate = (int)CM::Command::CommandSIGN,
-        Degrees = (int)CM::Command::CommandDegrees,
-        Pi = (int)CM::Command::CommandPI,
-        Sin = (int)CM::Command::CommandSIN,
-        Cos = (int)CM::Command::CommandCOS,
-        Tan = (int)CM::Command::CommandTAN,
-        Factorial = (int)CM::Command::CommandFAC,
-        XPower2 = (int)CM::Command::CommandSQR,
-        Mod = (int)CM::Command::CommandMOD,
-        FToE = (int)CM::Command::CommandFE,
-        LogBaseE = (int)CM::Command::CommandLN,
-        InvSin = (int)CM::Command::CommandASIN,
-        InvCos = (int)CM::Command::CommandACOS,
-        InvTan = (int)CM::Command::CommandATAN,
-        LogBase10 = (int)CM::Command::CommandLOG,
-        XPowerY = (int)CM::Command::CommandPWR,
-        YRootX = (int)CM::Command::CommandROOT,
-        TenPowerX = (int)CM::Command::CommandPOW10,
-        EPowerX = (int)CM::Command::CommandPOWE,
-        Exp = (int)CM::Command::CommandEXP,
-        DecButton = (int)CM::Command::CommandDec,
-        OctButton = (int)CM::Command::CommandOct,
-        HexButton = (int)CM::Command::CommandHex,
-        BinButton = (int)CM::Command::CommandBin,
-        And = (int)CM::Command::CommandAnd,
-        Ror = (int)CM::Command::CommandROR,
-        Rol = (int)CM::Command::CommandROL,
-        Or = (int)CM::Command::CommandOR,
-        Lsh = (int)CM::Command::CommandLSHF,
-        Rsh = (int)CM::Command::CommandRSHF,
-        Xor = (int)CM::Command::CommandXor,
-        Not = (int)CM::Command::CommandNot,
-        Sinh = (int)CM::Command::CommandSINH,
-        Cosh = (int)CM::Command::CommandCOSH,
-        Tanh = (int)CM::Command::CommandTANH,
-        InvSinh = (int)CM::Command::CommandASINH,
-        InvCosh = (int)CM::Command::CommandACOSH,
-        InvTanh = (int)CM::Command::CommandATANH,
-        Cube = (int)CM::Command::CommandCUB,
-        DMS = (int)CM::Command::CommandDMS,
+        // Enum values below are used for Tracelogging and do not map to the Calculator engine
+        MemoryAdd,
```

Comment:
We should set these to well-known integers--otherwise someone might accidentally change them in the future.